content/blog/magrittr-2-0/index.Rmd

```diff
@@ -39,6 +39,8 @@ This blog post covers the three main changes in this new version of the magrittr
 library(magrittr)
 ```
 
+__Update 2020-09-22:__ Some minor issues have arised from the first round of beta testing (see note on visibility and laziness below). We postponed the release to late October to give more time for potential further issues to surface before publication of magrittr 2.0 to CRAN.
```

Comment:
I think we also need a little on using `return()` in pipes, showing the error message that you'll see if you've used return inappropriately.

Reply:
Done in https://github.com/tidyverse/tidyverse.org/pull/468/commits/2370e6126c304981eb7c338ffd136d0ea465373b